### DISCUSSION TITLE
Access filtered / sorted data

### DISCUSSION THREAD

**Discussion by hemanzur** [vincjo/svelte-simple-datatables#65] [Q&A]
Hi, first of all thanks for your work. Very useful.
I'm using the library to sort / filter a bunch of data, and as I perform those actions I'd like to plot the data in a chart.
Is there any way to access the filtered data ?

Thanks

**Comment by vincjo** [comment]
Hi,

The ``$rows`` store contains the filtered data. But the problem is that ``$rows`` is possibly paginated (exposes only a slice of the records).

For the moment, the lib lacks a data manager. It will be added next September/October.
Unfortunately, there is no good solution in the meantime.

This use case will be taken into account for the next release
